spec/controllers/api/v1/companies_controller_spec.rb

```diff
@@ -0,0 +1,42 @@
+require 'rails_helper'
+
+RSpec.describe Api::V1::CompaniesController, type: :controller do
+
+  describe "GET #show" do
+    let(:response_body) { JSON.parse(response.body) }
+    let(:company) { create(:company) }
+
+    it "is successful" do
+      get :show, id: company.id, format: :json
+
+      expect(:success)
+    end
+
+    it 'returns company with correct attributes' do
+      get :show, id: company.id, format: :json
+      json_company = response_body['company']
+
+      expect(json_company['name']).to be_instance_of(String)
+      expect(json_company['jobs']).to be_instance_of(Array)
+    end
```

Comment:
I agree. I found @theonlyrao's wording informative, but not cautioning enough for me: when you robustify this test, avoid adding any serializer dependency to it.